### PR TITLE
Fix build errors

### PR DESCRIPTION
**Pull request by lokeshbv**
When configured with `qemu/tc_64k` defconfig, following build errors occur,

```
vfs/fs_mkdir.c: In function 'mkdir':
vfs/fs_mkdir.c:164:4: error: implicit declaration of function 'memset'; did you mean 'sigset'? [-Werror=implicit-function-declaration]
vfs/fs_mkdir.c:165:4: error: implicit declaration of function 'strncpy' [-Werror=implicit-function-declaration]

le_tc/filesystem/fs_main.c: In function 'tc_fs_vfs_ioctl':
le_tc/filesystem/fs_main.c:1598:29: error: passing argument 3 of 'ioctl' makes integer from pointer without a cast [-Werror=int-conversion]

le_tc/filesystem/fs_main.c: In function 'tc_fs_vfs_ioctl':
le_tc/filesystem/fs_main.c:1596:8: error: implicit declaration of function 'ioctl'; did you mean 'prctl'? [-Werror=implicit-function-declaration]

le_tc/filesystem/fs_main.c: In function 'tc_libc_stdio_lib_rdflush':
le_tc/filesystem/fs_main.c:2359:8: error: implicit declaration of function 'lib_rdflush'; did you mean 'lib_noflush'? [-Werror=implicit-function-declaration]

le_tc/filesystem/fs_main.c:284:8: error: implicit declaration of function 'mount' [-Werror=implicit-function-declaration]

le_tc/filesystem/fs_main.c: In function 'tc_fs_vfs_umount':
le_tc/filesystem/fs_main.c:306:8: error: implicit declaration of function 'umount' [-Werror=implicit-function-declaration]
```

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [c22d9bc31d0a612fd1290b3bccd7d4deb71963a1] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c22d9bc31d0a612fd1290b3bccd7d4deb71963a1] - Code Rule Check OK.

File: apps/examples/testcase/le_tc/filesystem/fs_main.c [hunk 1]
```diff
@@ -50,6 +52,7 @@
 #include <time.h>
 #include "tc_common.h"
 #include "tc_internal.h"
+#include "../../../../../lib/libc/lib_internal.h"
```
[the remark this answers is not in the document]
Comment (lokeshbv) [reply apps/examples/testcase/le_tc/filesystem/fs_main.c]: lib_rdflush is used in fs tc. Prototype of lib_rdflush is present in lib_internal.h

**Comment by seinfra** [comment]
Target : [45a0dd00933fecf11d91c304f890d1d28bad8231] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [45a0dd00933fecf11d91c304f890d1d28bad8231] - Code Rule Check OK.

**Pull request merged**